core/variables.tf

```diff
@@ -26,3 +26,8 @@ variable "redis_auth_token" {
   description = "The auth token for the Redis cluster"
   type        = string
 }
+
+variable "bastion_allowed_ip_connection" {
+  description = "IP that can be connected to Bastion instance"
+  type        = string
+}
```

Comment:
small thing, it should be better to define the variable as a list, so more IPs can be whitelisted if needed.

Reply:
also it can be moved to local variables, there is no need to keep it in tf variables

Reply:
It was updated to be a local variable in e6aab02 and changed to be a list of strings as well. Thank you!!!!